cmd/config/checks.go

```diff
@@ -59,6 +60,15 @@ func setCheckValidRepos(check voucher.Check, validRepos []string) {
 	}
 }
 
+// setCheckTrustedBuilderIdentities sets trusted builder identities and projects used in ProvenanceCheck
+// trustedBuilderIdentities is a map of BuildCreators to bool, which is set to true for trusted build creators
+// trustedBuilderProjects is a map of projects to bool, which is set to true for trusted projects
+func setCheckTrustedBuilderIdentities(check voucher.Check, trustedBuilderIdentities map[string]bool, trustedBuilderProjects map[string]bool) {
```

Comment:
This is good! Could we rename it to:
```suggestion
func setCheckTrustedIdentitiesAndProjects(check voucher.Check, trustedBuilderIdentities []string, trustedBuilderProjects []string) {
```

Since we're not just trusting identities, we're also trusting projects here!

Reply:
Done.